### ISSUE TITLE
WN18RR result of GenKGC

### ISSUE THREAD

**Issue by gyf6666** [zjunlp/PromptKG#55]
My results on the wn18rr dataset are particularly low, with more than ten points missing for each metric. I 've seen this problem in the past, but I can 't find PromptKG/toolkit
![image](https://github.com/user-attachments/assets/53776ce6-3b08-4223-a33a-848c77379e3d)
![image](https://github.com/user-attachments/assets/bca246c2-6948-4205-8ce4-c1b23c097dc6)
![image](https://github.com/user-attachments/assets/6b264e27-a47f-4b7e-98c3-afb2ac67b819)

**Comment by CheaSim** [comment]
You can adjust the hyperparameters in the experiment and I suggest larger batch size and more epochs to train the model.

**Comment by zxlzr** [comment]
Hi, have you solved your issue yet?

**Comment by gyf6666** [comment]
Thank you. I succeeded in your way.
